rootfs/etc/nginx/lua/balancer/sticky.lua

```diff
@@ -22,6 +22,7 @@ function _M.new(self, backend)
     digest_func = digest_func,
     traffic_shaping_policy = backend.trafficShapingPolicy,
     alternative_backends = backend.alternativeBackends,
+    original_cookie_session_affinity = backend["sessionAffinityConfig"]["cookieSessionAffinity"]
```

Comment:
Just call it `cookie_session_affinity` to match with `cookieSessionAffinity`.

Reply:
fixed